test/e2e/specs/wp-plan-purchase-spec.js

```diff
@@ -56,6 +56,22 @@ describe( `[${ host }] Plans: (${ screenSize }) @parallel @jetpack`, function()
 			return await plansPage.waitForComparison();
 		} );
 
+		step( 'Can See Exactly One Primary CTA Button', async function() {
+			const plansPage = await PlansPage.Expect( driver );
+			return assert(
+				await plansPage.onePrimaryButtonShown(),
+				'Incorrect number of primary buttons'
+			);
+		} );
+
+		step( 'Can See Exactly One Primary CTA Button for Mobile', async function() {
```

Comment:
By combining the functions above, we can remove the second  check for mobile here.